vars/main.yml

```diff
@@ -0,0 +1,29 @@
+restund_all_public_ipv4s: "{{ ansible_facts.all_ipv4_addresses | ipaddr('public') }}"
+restund_all_private_ipv4s: "{{ ansible_facts.all_ipv4_addresses | ipaddr('private') }}"
+
+# INFO: Lists of private IP ranges to help setting up firewall rules and prevent exposure of private services
+# DOCS: https://www.rtcsec.com/post/2021/01/details-about-cve-2020-26262-bypass-of-coturns-default-access-control-protection/#further-concerns-what-else
+restund_blocked_ipv4_network_ranges:
```

Comment:
This should be moved to `defaults` not `vars`.  Many customers are actually deploying Wire inside a VPN and there they only _have_ private address ranges. So they should be able to override this

Reply:
Ah you have a `restund_allowed_private_network_cidrs` later in the PR. I see